ui_espace_pro/src/pages/Candidat/FormCreatePage.jsx

```diff
@@ -311,7 +382,7 @@ export const FormCreatePage = (props) => {
                   <Button
                     variant="unstyled"
                     type={"submit"}
-                    loading={submitLoading}
+                    loading={submitLoading.toString()}
```

Comment:
```suggestion
                    loading={submitLoading}
```

submitLoading est un boolean

Reply:
submitLoading n'est pas supporté en tant que booléen sur la propriété loading